### PR TITLE
Add more fields to Manifest form

### PR DESCRIPTION
**Pull request by jessieay**
- Slowly but surely, we're getting there

screen shot 2016-01-12 at 5 30 15 pm

### CODE REVIEW

File: spec/features/create_manifest_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 feature 'Create manifest' do
-  scenario 'successfully' do
+  scenario 'fills fills in tracking number' do
```
Comment (pkarman) [inline spec/features/create_manifest_spec.rb]: typo?

**Comment by pkarman** [comment]
@jessieay merge conflict, alas

**Comment by jessieay** [comment]
@pkarman rebased!

File: spec/features/create_manifest_spec.rb (hunk 1 above)
Comment (jessieay) [inline spec/features/create_manifest_spec.rb]: what? you haven't heard of fills fills?
 jk, total typo. fixing

**Pull request merged**